src/components/pages/about-us/team.tsx

```diff
@@ -11,7 +11,7 @@ const TeamLayout = styled.div`
   margin-top: 48px;
 
   display: grid;
-  grid-template-columns: repeat(auto-fit, minmax(160px, 1fr));
+  grid-template-columns: repeat(auto-fit, minmax(150px, 1fr));
```

Comment:
Ensures that the team member grid has two columns also on small mobile devices (e.g. iPhone SE)

Reply:
We might lower that even more to have two columns on any mobile device. Which means to target device width of 320px. Right now it breaks into one column at 356px because of the space taken by the grid container and the gap it wants between the images. We probably want (320 - 48 - 8)/2 = 132px as the lower bounds ( for your convenience, I was in the mood to check on the numbers)

![Screenshot 2023-03-10 at 08 48 32](https://user-images.githubusercontent.com/1701755/224255254-1b44ba34-6116-4e64-9b5c-ae7ac39d0c8d.png)
![Screenshot 2023-03-10 at 08 48 28](https://user-images.githubusercontent.com/1701755/224255246-d8fdca7b-8a40-49db-a418-ce0c6bb04a52.png)